content/applications/finance/fiscal_localizations/brazil.rst

```diff
@@ -0,0 +1,327 @@
+
+   You can add or delete accounts according to the company's needs.
+
+Taxes
+~~~~~
+
+Taxes are automatically created when installing the Brazilian localization. Taxes are already
+configured, and some of them are used by Avalara when computing taxes on the sales order or invoice.
+
+Taxes can be edited or more taxes can be added. For example, some taxes used for services need to be
+manually added and configured, as the rate may differ depending on the city where you are offering
+the service.
+
+.. important::
+   Taxes attached to services are not computed by AvaTax. Only goods taxes are computed.
+
+When configuring a tax used for a service that is included in the final price (when the tax is not
+added or subtracted on top of the product price), set the :guilabel:`Tax Computation` to
+:guilabel:`Percentage of Price Tax Included`, and, on the :guilabel:`Advanced Options` tab, check
+the :guilabel:`Included in Price` option.
+
+For more information on how to configure taxes to better fit your needs, please go to the
+:doc:`taxes functional documentation </applications/finance/accounting/taxes>`.
+
+.. image:: brazil/tax-configuration.png
+   :alt: Tax configuration.
+
+.. warning::
+   Do not delete taxes, as they are used for the AvaTax tax computation. If deleted, Odoo creates
+   them again when used in an |SO| or invoice and computing taxes with AvaTax, but the account used
+   to register the tax needs to be re-configured in the tax's :guilabel:`Definition` tab, under the
+   :guilabel:`Distribution for invoices` and :guilabel:`Distribution for refunds` sections.
+
+.. _brazil/products:
+
+Products
+~~~~~~~~
+
+To use the AvaTax integration on sale orders and invoices, first specify the following information
+on the product:
+
+- :guilabel:`CEST Code`
+- :guilabel:`Mercosul NCM Code`
+- :guilabel:`Source of Origin`
+- :guilabel:`SPED Fiscal Product Type`
+- :guilabel:`Purpose of Use`
+- :guilabel:`Sales Price`
+
+.. image:: brazil/product-configuration.png
+   :alt: Product configuration.
+
+.. note::
+   Odoo automatically creates three products to be used for transportation costs associated with
+   sales. These are named `Freight`, `Insurance`, and `Other Costs`, they are already configured, if
+   more need to be created, duplicate and use the same configuration (configuration needed:
+   :guilabel:`Product Type` `Service`, :guilabel:`Transportation Cost Type` `Insurance`,
+   `Freight`, or `Other Costs`)
+
+.. _brazil/contacts:
+
+Contacts
+~~~~~~~~
+
+Before using the integration, specify the following information on the contact:
+
+#. General information about the contact:
+
+   - Select the :guilabel:`Company` option for a contact with a tax ID (CNPJ), or check
+     :guilabel:`Individual` for a contact with a CPF
+   - :guilabel:`Name`
+   - :guilabel:`Address`: :guilabel:`Zip Code` is a required field to properly compute taxes.
+   - :guilabel:`Tax ID` or :guilabel:`CPF`: depending on the case complete with the number
```

Comment:
unclear

Reply:
```suggestion
   - :guilabel:`Tax ID` or :guilabel:`CPF`: Use CPF for individuals and Tax ID (CNPJ) for companies
```
Also, in BR users know what these abbreviations refer to, but true that this is unclear.
@samueljlieber can you help me with this change?
Thanks!